src/daemon/krillserver.rs

```diff
@@ -604,7 +604,7 @@ impl KrillServer {
         self.ca_manager.get_ca(handle).await.map(|ca| ca.publisher_request())
     }
 
-    pub async fn ca_init(&mut self, init: CertAuthInit) -> KrillEmptyResult {
+    pub async fn ca_init(&self, init: CertAuthInit) -> KrillEmptyResult {
```

Comment:
I don’t think this method needs to be async?

Reply:
indeed, removed